src/PowerShellEditorServices/Utility/ObjectPool.cs

```diff
@@ -0,0 +1,19 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using System.Collections.Concurrent;
+
+namespace Microsoft.PowerShell.EditorServices.Utility
+{
+    public class ObjectPool<T>
```

Comment:
The build is currently failing because XML comments are missing for these types.

`ObjectPool` should be `internal` I imagine, but it would also be worthwhile to comment them to explain their basic semantics if you've got time.

Reply:
Also, the fact that the XML comments fails a Release build but not a Debug build is something we need to fix :)

Reply:
Yeah, I hit similar problems as well when I was building on Linux.

I just never think to run a release build and then CI fails for something I wish I'd at least been warned about locally...

Anyway, the work to move off net451 has already been done downstream in the v2 branch, so soon enough it will be easy for Linux and macOS users to have exactly the same build experience as Windows.